docs/docs/basics/getting-started.md

```diff
@@ -4,19 +4,8 @@ sidebar_position: 2
 
 # Getting Started
 
-First of all, you need to set up the player. This usually takes less than a
-second:
-
-```ts
-import TrackPlayer from 'react-native-track-player';
-
-await TrackPlayer.setupPlayer({})
-// The player is ready to be used
-```
-
-You also need to register a [playback service](./playback-service.md) right after
-registering the main component of your app:
-
+## Starting off
+First, you need to register a [playback service](#playback-service) right after registering the main component of your app (typically in your `index.js` file at the root of your project):
```

Comment:
I believe we should be pointing at `./playback-service.md` instead of `#playback-service`.

Reply:
Ah forgot about that, I'll switch it back